R/annotateProteinInfoFromIndra.R

```diff
@@ -0,0 +1,115 @@
+#' Annotate Protein Information from INDRA
```

Comment:
Could you add a simple unit test for this file?  Also with a scenario with uniprot ID cannot be converted too to verify things go smoothly.

Reply:
done